dpub-aria/index.html

```diff
@@ -38,36 +41,20 @@
             name: "Shane McCarron",
             url: "http://blog.halindrome.com",
             company: "Spec-Ops",
-            companyURL: "https://www.spec-ops.io",
+            companyURL: "https://daisy.org/https://spec-ops.io/",
```

Comment:
Looks like an accidental paste of the daisy url happened here.
```suggestion
            companyURL: "https://spec-ops.io/",
```